deploy/webhook.yaml

```diff
@@ -0,0 +1,264 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: service-apis
+  labels:
+    name: service-apis
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: service-apis-admission
+webhooks:
+  - name: validate.networking.x-k8s.io
+    matchPolicy: Equivalent
+    rules:
+      - operations: [ "CREATE" , "UPDATE" ]
+        apiGroups: [ "networking.x-k8s.io" ]
+        apiVersions: [ "v1alpha1" ]
+        resources: [ "httproutes" ]
+    failurePolicy: Fail
+    sideEffects: None
+    admissionReviewVersions:
+      - v1
+      - v1beta1
+    clientConfig:
+      service:
+        name: service-apis-admission
+        namespace: service-apis
+        path: "/validate"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: service-apis-admission
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
+  namespace: service-apis
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: service-apis-admission
+  labels:
+    name: service-apis
+    version: 0.0.1
+rules:
+  - apiGroups:
+      - admissionregistration.k8s.io
+    resources:
+      - validatingwebhookconfigurations
+    verbs:
+      - get
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: service-apis-admission
+  annotations:
+  labels:
+    name: service-apis-webhook
+    version: 0.0.1
```

Comment:
What is the purpose of this version field?
I'm concerned this would get outdated since it is not automated.